src/libraries/System.ComponentModel.TypeConverter/src/System/ComponentModel/TypeDescriptor.cs

```diff
@@ -2919,10 +2926,11 @@ private sealed class ComNativeDescriptorProxy : TypeDescriptionProvider
         {
             private readonly TypeDescriptionProvider _comNativeDescriptor;
 
+            [UnconditionalSuppressMessage("ReflectionAnalysis", "IL2072:UnrecognizedReflectionPattern",
+                Justification = "The trimmer can't find the ComNativeDescriptor type because System.Windows.Forms isn't available.")]
```

Comment:
Technically this creates an external dependency on the `ComNativeDescriptor` - I guess we rely on the fact that if the app uses WinForms then trimmer will actually see the type and preserve it accordingly, right?
This might be worth a comment.

Reply:
> we rely on the fact that if the app uses WinForms then trimmer will actually see the type and preserve it accordingly, right?

That was my thinking.

> This might be worth a comment.

I changed the justification to be:

```C#
[UnconditionalSuppressMessage("ReflectionAnalysis", "IL2072:UnrecognizedReflectionPattern",
    Justification = "The trimmer can't find the ComNativeDescriptor type when System.Windows.Forms isn't available. " +
    "When System.Windows.Forms is available, the type will be seen by the trimmer and the ctor will be preserved.")]
```